plugins/meson.build

```diff
@@ -1,5 +1,6 @@
 subdir('dfu')
 subdir('ebitdo')
+subdir('flashrom')
```

Comment:
I think it would be good to make this plugin optional by meson config option (but default on).  Eg thinking with a RHEL hat on, I'm not sure it's something that should be on RHEL without an explicit ask from a customer.

Reply:
What's the logic for not including it in RHEL? Perhaps that the flashrom update mechanism is inherently riskier than a UEFI capsule update?

Reply:
Yeah inherently riskier is exactly what I was getting at. There's no validation of the signature of a payload.

Reply:
Right, agree. Is flashing an unsigned system firmware from the LVFS more risky than not applying a security update? My argument would be you remove the layer of hardware validation, and fall back to just the two layers of LVFS+fwupd policy.

Reply:
I suppose and if flashrom the runtime dep isn't installed by deafult yet this is an extremely small surface area.